int32 unique_id

TrackedPoint pose
